app/src/containers/RegisterPage/index.js

```diff
@@ -203,6 +205,24 @@ class SignUpPage extends Component {
     this.setState({ email: value });
   };
 
+  handleChangeConfirmEmail = event => {
+
+    var value = (event.target.value);
+    this.setState({ emailConfirmation: value });
+  
+
+   if (value !== this.state.email) {
+      this.setState({ emailConfirmationError: true, emailConfirmationErrorMessage: 'The email and confirm email must be equal' });
+      this.setState({ confirmEmailChecked: false });
+    } else {
```

Comment:
why do you need so many properties in state ?? What is the difference between `emailConfirmationError ` and `confirmEmailChecked`? Why cannot use either one of those 

this.SetState can be combined in on one call, just add more property into the obj, please research more about this.setState

`confirmEmailChecked ` this is a static string, why do we need to use state for this ? 

Error text should be  `The email and confirm email must be the same`